TestProject/input/Base.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace input;
+
+use options\Getopt;
+use cellularAutomat\Board;
+
+/**
+ * Base class for pluggable inputs.
+ *
+ * Fills the created board with the given height and width values.
+ *
+ * Adds additional options for pluggable input.
+ *
+ * @class BaseInput
+ */
+abstract class Base
+{
+    /**
+     * Adds additional option which can be called over input.
+     *
+     * @param Getopt $_options
+     */
+    function addOptions(Getopt $_options)
+    {
+    }
+
+    /**
+     * Catches the created board.
```

Comment:
What do you mean with catches? What about "Allows the input to fill the given \c $_board as the input wants."